dev-support/docker-images/jupyter/Dockerfile

```diff
@@ -91,8 +91,13 @@ RUN wget --quiet https://repo.continuum.io/miniconda/Miniconda3-py37_${MINICONDA
     conda clean --all -f -y && \
     rm -rf /home/$NB_USER/.cache/yarn
 
-# Install sumbarine python sdk and notebook
-RUN pip --quiet --no-cache-dir install notebook==6.1.3 apache-submarine
+# Install latest sumbarine python sdk and notebook
+RUN pip install notebook==6.1.3 && \
+    git clone https://github.com/apache/submarine && \
```

Comment:
Do we need to support build with different versions of SDK?

Reply:
We need to install the lastest pysubmarine when building this dockerfile, it's easy for testing SDK in a notebook.
If we pull `apache/submarine:jupyter-notebook-0.5.0`, the version of SDK in the container still is 0.5.0